qcfractal/interface/collections/openffworkflow.py

```diff
@@ -202,7 +202,7 @@ def add_fragment(self, fragment_id, data, provenance=None):
 
             # add back to fragment data
             packet["hash_index"] = ret
-            packet["provenance"] = provenance
+            # packet["provenance"] = provenance
```

Comment:
Will we want to add this back in when we have the `Provenance` model?

Reply:
No, thinking on it I don't think so here. The provenance information will be overridden by TorsionDrive itself here.

Reply:
We will need to reorganize somethings here so that this is `fragment` provenance data not TD provenance data. I will spin out an issue.

Reply:
Thats fine, do we want to fix it as part of this PR or a future one?

Reply:
Future, writing an issue now.